### DISCUSSION TITLE
ISSUE [Answered] (User didn't read the guide)

### DISCUSSION THREAD

**Discussion by boanoitepessoas** [Refloow/Steam-Card-Bot-PRO#4] [Q&A] [answered]
Everything that is calling about finding out the other person's level or calculating the level is giving an error try to give            
      "!level [value]

**Comment by Refloow** [comment, marked as answer]
You should report issues in project Issues not in the Discussion. 
Everything works alright. For bot to fetch user data make sure to provide an valid Steam API (one from bots account).
More about it find in the setup wiki guide

Login into the browser and you can get the code from
https://steamcommunity.com/dev/apikey
